templates/taxi/car_confirm_delete_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Removing the car model from Taxi service</h1>
+  <p>
+    Are you sure that you want to delete {{ car }}
+    from Taxi service?
+  </p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit"
+           value="Yes, delete"
+           class="btn btn-danger"
+    >
+  </form>
+{% endblock %}
```

Comment:
please, add a blank for the end of the file

Reply:
ok, my mistake..